__tests__/topics.test.js

```diff
@@ -0,0 +1,46 @@
+const request = require("supertest");
+
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("responds with array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug", expect.any(String));
+          expect(topic).toHaveProperty("description", expect.any(String));
+        });
+      });
```

Comment:
Really good test using a forEach and not expecting the exact object back, this allows for later expansion :)